### PR TITLE
Changelog: initial change for 8.9 release

### PR DESCRIPTION
**Pull request by jeherve**
#### Changes proposed in this Pull Request:

* Changelog for the 8.9 release

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* N/A

#### Testing instructions:

* Check for typos.

#### Proposed changelog entry for your changes:

* This is it.

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -96,82 +96,63 @@ There are opportunities for developers at all levels to contribute. [Learn more
...
-* Markdown block: improve handling of long links with custom characters, as well as em- and en- dashes.
-* Payments block: add extra customization options to the Submit button.
-* Performance: use WordPress-provided wp_resource_hints for DNS prefetching.
-* Podcast Player block: start saving content in post content for better compatibility with non-WordPress tools.
-* Sharing: add direct link to share button customization for logged-in admins.
-* Widgets: introduce new Instagram Widget.
-* Widgets: add additional options to the Twitter Timeline widget.
-* Widgets: add the rel attribute to links with target=”_blank” in the Social Icons widget.
-* WordAds: update ad units to be more flexible and dynamic.
+* Blocks: add "Edit" button the Calendly block toolbar.
```
Comment (mdbitz) [inline readme.txt:111]: *to* the Calendly block

**Review by mdbitz — DISMISSED** [review]
@jeherve only noticed the need for the 2 *to*s :)

File: changelog.txt [hunk 2]
```diff
@@ -2,6 +2,64 @@
...
+* Release post: https://wp.me/p1moTy-rAs
+
+**Major Enhancements**
+
+* The new Donations Block allows you to easily accept and process donations on your site.
+* You can now preview how your posts will look like on Facebook, Twitter, and Google Search Results even before you hit the Publish button.
+
+**Enhancements**
+
+* Blocks: add "Edit" button the Calendly block toolbar.
```
[the remark this answers is not in the document]
Comment (jeherve) [reply changelog.txt]: Good catch, fixed.

File: readme.txt (hunk 1 above)
Comment (jeherve) [reply readme.txt]: 👍

File: changelog.txt [hunk 3]
```diff
@@ -2,6 +2,64 @@
...
 
+= 8.9 =
+
+* Release date: September 1, 2020
+* Release post: https://wp.me/p1moTy-rAs
+
+**Major Enhancements**
+
+* The new Donations Block allows you to easily accept and process donations on your site.
+* You can now preview how your posts will look like on Facebook, Twitter, and Google Search Results even before you hit the Publish button.
```
Comment (kbrown9) [inline changelog.txt:13]: What do you think of removing 'like'?

```suggestion
* You can now preview how your posts will look on Facebook, Twitter, and Google Search Results even before you hit the Publish button.

```

**Pull request merged**